lnbits/static/js/users.js

```diff
@@ -351,12 +351,14 @@ new Vue({
       console.log('export users')
     },
     topupCallback(res) {
-      this.wallets.forEach(wallet => {
-        if (res.wallet_id === wallet.id) {
-          wallet.balance_msat += res.value * 1000
-        }
-      })
-      this.fetchUsers()
+      if (res.success) {
+        this.wallets.forEach(wallet => {
```

Comment:
why is there a foreach here?

Reply:
i want to update the balance inside the list and this.wallets is the reactive property so i iterate through them to change the balance there